tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -668,3 +219,331 @@ def _run(self):
                 payload={"message": "%s complete" % self.__class__.__name__}
             )
         )
+
+
+def emit_event(self, resource, curr_value, msg, instance):
```

Comment:
Yes. Currently it is based on changes in `status` value. Would add events based on `state` change as well here.